### PR TITLE
gh-106392: Fix inconsistency in deprecation warnings in datetime module

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
* Issue: gh-106392

### CODE REVIEW

File: Lib/_pydatetime.py [hunk 1]
```diff
@@ -1827,7 +1827,7 @@ def now(cls, tz=None):
     def utcnow(cls):
         "Construct a UTC datetime from time.time()."
         import warnings
-        warnings.warn("datetime.utcnow() is deprecated and scheduled for "
+        warnings.warn("datetime.datetime.utcnow() is deprecated and scheduled for "
                       "removal in a future version. Instead, Use timezone-aware "
```
Comment (hauntsaninja) [inline Lib/_pydatetime.py]: Fix capitalisation / consistency with C messages
```suggestion
                      "removal in a future version. Use timezone-aware "
```

**Review by hauntsaninja — APPROVED** [review]

**Comment by hauntsaninja** [comment]
Thank you for fixing!

**Comment by serhiy-storchaka** [comment]
Thank you for your review!

**Comment by neutrinoceros** [comment]
For reference, this PR was a dupe to #109080 , but I understand it's hard to keep track of everything. Thanks for fixing it !

**Pull request merged**